packages/@aws-cdk/aws-route53/lib/alias-record-target.ts

```diff
@@ -11,6 +12,21 @@ export interface IAliasRecordTarget {
   bind(record: IRecordSet): AliasRecordTargetConfig;
 }
 
+/**
+ * Validatable alias record target class
+ */
+export interface IValidatableAliasRecordTarget extends IAliasRecordTarget {
```

Comment:
awslint seems to be complaining about the interface not being used, while I implement it in `bucket-website-target`:

```
@aws-cdk/aws-route53: error: [awslint:no-unused-type:@aws-cdk/aws-route53.IValidatableAliasRecordTarget] type or enum is not used by exported classes (declared at lib/alias-record-target.ts:18)
```

Reply:
@nmussy did you need a hand with this one - I can pull it down to take a look at what awslint might be complaining about